domain/schema/controller_schema_test.go

```diff
@@ -196,10 +196,18 @@ func (s *controllerSchemaSuite) TestControllerTriggers(c *gc.C) {
 		"trg_log_cloud_credential_update",
 		"trg_log_cloud_credential_delete",
 
+		"trg_log_cloud_credential_attribute_insert",
```

Comment:
Can you put these guys after `trg_log_cloud_ca_cert`* so my OCD can sleep tonight? :)

Reply:
I wanted to keep the cloud and cloud credential related things grouped together.